misc/multiaddr/src/lib.rs

```diff
@@ -120,6 +120,11 @@ impl fmt::Display for Multiaddr {
 }
 
 impl Multiaddr {
+    /// Create a new, empty multiaddress.
+    pub fn empty() -> Multiaddr {
```

Comment:
`#[inline]`

Reply:
What about: https://internals.rust-lang.org/t/when-should-i-use-inline/598/3 ?

Reply:
That is from 2014 and I think is no longer true.